index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<head>
+    <title>Sobia Furqan's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+<nav>
+    <section>
+        <div class="nav-links">
+            <a href="#home" class="home">Home</a>
+            <a href="#about-me" class="about-me">About Me</a>
+            <a href="#projects" class="work">Projects</a>
+            <a href="#contact" class="contact">Contact</a>
+    </section>
+    </div>
```

Comment:
These two closing tags are the wrong way around, the closing div should come before the section. They always get closed in the order they were opened :)